src/services/jwt.service.js

```diff
@@ -0,0 +1,52 @@
+import jwt from 'jsonwebtoken';
+
+const generateAccessToken = (user) => {
+  return jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '10m',
+  });
+};
+
+const generateRefreshToken = (user) => {
+  return jwt.sign(user, process.env.JWT_REFRESH_KEY, {
+    expiresIn: '10m',
+  });
+};
+
+const validateAccessToken = (token) => {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch (e) {
+    return null;
+  }
+};
+
+const validateRefreshToken = (refreshToken) => {
+  try {
+    return jwt.verify(refreshToken, process.env.JWT_REFRESH_KEY);
+  } catch (e) {
+    return null;
+  }
+};
+
+const generateResetPasswordToken = (user) => {
+  return jwt.sign(user, process.env.JWT_RESET, {
+    expiresIn: '5m',
```

Comment:
The reset password token expiration time is set to '5m'. Ensure this duration is appropriate for your application's password reset process.